env/env.go

```diff
@@ -0,0 +1,49 @@
+package env
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+var (
+	configuration *Static
+	ConfigFlag    string = "DOWNDETECTOR_CONFIG"
+)
+
+type Static struct {
+	BotToken    string `json:"bot_token"`
+	BotGuild    string `json:"bot_guild"`
+	Address     string `json:"address"`
+	ChannelName string `json:"channelName"`
+	Checks      []Check
+}
+
+type Check struct {
+	Type     string `json:"type"`
+	Value    string `json:"value"`
+	Interval string `json:"interval"`
+}
+
+func Configuration() *Static {
```

Comment:
Implement this function like this: https://github.com/infiniteloopcloud/discord-jira/blob/main/env/env.go#L22 Because the multiple level of assignment of variable s is not a good idea.